modules/income_and_assets/lib/income_and_assets/pdf_fill/forms/va21p0969.rb

```diff
@@ -843,7 +762,15 @@ class Va21p0969 < ::PdfFill::Forms::FormBase
             question_text: 'WAIVED GROSS MONTHLY INCOME'
           }
         }
-      }.freeze
+      }
+
+      # NOTE: Adding these over the span of multiple PRs too keep the LOC changed down.
+      # Going to add them in reverse order so that the keys maintain the previous ordering
+      SECTIONS = [Section11].freeze
+
+      SECTIONS.each { |section| key = key.merge(section::KEY) }
```

Comment:
There is probably a way to collect these into the `KEY` constant directly, something like: `KEY = ...`